src/libData/AccountData/AccountStore.cpp

```diff
@@ -897,6 +899,19 @@ bool AccountStore::MigrateContractStates(
     } else {
       numContractUnchangedStates++;
     }
+
+    if (count % updateDiskFrequency == 0) {
+      if (!UpdateStateTrieAll()) {
+        LOG_GENERAL(WARNING, "UpdateStateTrieAll failed");
+        return false;
+      }
+
+      /// repopulate trie and discard old persistence
+      if (!MoveUpdatesToDisk()) {
```

Comment:
Seems like build error is complaining about this part
https://app.travis-ci.com/github/Zilliqa/Zilliqa/builds/236072281

Maybe some commits should be merged before this?